main.py

```diff
@@ -25,25 +31,62 @@ def value(self, value):
 
 
 class Name(Field):
-    pass
+    def __init__(self, n):
+        self.name = n
```

Comment:
No need to make additional init as init is already added in parent(Field) class. Better to leave "pass"